lib/controls/ValueSelector.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class ValueSelector extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {value, options, className, handleOnChange} = this.props;
+    const children = this.props.children;
+
+    if (children) {
+      return (
+        React.cloneElement(children,
```

Comment:
Not entirely sure what you mean here - I was hoping to check for a child element and render if it exists, otherwise fallback to the default. I can definitely remove the `shouldRender` from the `ValueEditor` but is there a better pattern to achieve the "fallback" effect?